beets/Dockerfile

```
@@ -1,4 +1,4 @@
-FROM python:2
+FROM python:3.8-alpine
```

Seems fine (https://github.com/beetbox/beets/blob/v1.4.9/setup.py#L171-L176), just think we should take the opportunity to pin better:

```suggestion
FROM python:3.8-alpine3.10
```